flocx_market/api/bid.py

```diff
@@ -0,0 +1,44 @@
+from flask_restful import Resource
+from flocx_market.db.sqlalchemy.bid_api import BidApi
+from flask import request
+
+
+class Bid(Resource):
+
+    @classmethod
+    def get(cls, marketplace_bid_id):
```

Comment:
This will fail for `GET /bid` because `marketplace_bid_id` is a required parameter.  Change it to a keyword parameter with a default value:

```
def get(cls, marketplace_bid_id=None):
```

Also, this is why we write tests :). The error shows up very quickly if you test the behavior against the specification.